src/XrdHttpTpc/XrdHttpTpcConfigure.cc

```diff
@@ -96,6 +96,22 @@ bool TPCHandler::Configure(const char *configfn, XrdOucEnv *myEnv)
             } else {
                 m_first_timeout = 2*m_timeout;
             }
+        } else if (!strcmp("tpc.max_active_transfers_per_queue", val)) {
+            if (!(val = Config.GetWord())) {
+                Config.Close();
+                m_log.Emsg("Config", "tpc.max_active_transfers_per_queue value not specified.");
+                return false;
+            }
+            unsigned int max_active = std::stoi(val);
```

Comment:
I think this can switch to `XrdOuca2x::a2i`.  It's undocumented but here's the signature:

```
int XrdOuca2x::a2i(XrdSysError &Eroute, const char *emsg, const char *item,
                                             int *val, int minv, int maxv)
```

The `minv` and `maxv` arguments are "minimum value" and "maximum value" which is likely useful here.  It also takes care of logging of error messages, which may shorten the code a bit.